app/views/ats/positions/_profile.html.slim

```diff
@@ -10,45 +10,54 @@
   = render "ats/positions/side_header"
 / TAB LIST
 - content_for :tab_list
+  ruby:
```

Comment:
adapted like in candidate's profile
also solved problem with tasks counter that did not displayed in mobile version